src/ProxyClient/Varnish.php

```diff
@@ -109,7 +109,7 @@ public function invalidateTags(array $tags)
      */
     public function getTagsHeaderValue(array $tags)
     {
-        return implode(',', array_unique($this->escapeTags($tags)));
+        return array_unique($this->escapeTags($tags));
```

Comment:
A comma separated list is the standard HTTP way of having multiple values for a header. That applies here: we have multiple tags for the tag header.